packages/extension-browser/src/devtools/utils/analytics.ts

```diff
@@ -1,26 +1,46 @@
-import { AppInsights } from 'applicationinsights-js';
+import { ApplicationInsights } from '@microsoft/applicationinsights-web-basic';
+
+const instrumentationKey = '8ef2b55b-2ce9-4c33-a09a-2c3ef605c97d';
+
+let appInsights: ApplicationInsights;
+
+const trackEvent = (name: string, properties?: { [key: string]: string }, measurements?: { [key: string]: number }) => {
+    if (!appInsights) {
+        return;
+    }
+
+    appInsights.track({
+        baseData: {
```

Comment:
Not sure what `baseData` can have but I think we should indicate somehow where the extension is running (Edge, Chrome, Firefox...).  That will help us know if we have reliability issues in one more than another and which one is more popular.

Reply:
No need in this case. Application Insights already includes data about the browser and OS in submissions (though its logic does need to be updated to account for the new Edge).

Reply:
Nice